draft-ietf-ppm-dap.md

```diff
@@ -888,116 +888,42 @@ VDAF types:
 
 ## Batch Modes, Batches, and Queries {#batch-mode}
 
-Aggregated results are computed based on sets of reports, called "batches". The
-Collector requests the aggregated results for a given batch via a "query". The
-Aggregators use this query to carry out the aggregation flow and produce
-aggregate shares encrypted to the Collector.
+An aggregate result is computed from a sets of reports, called a "batch". The
+Collector requests the aggregate result for a given batch via a "query". The
+Aggregators use this query to select a batch for aggregation.
 
 Each measurement task has a preconfigured "batch mode". The batch mode defines
 both how reports may be partitioned into batches, as well as how these batches
 are addressed and the semantics of the query used for collection.
 
-This document defines the time-interval and leader-selected batch modes.
-Batch modes are identified with a single byte in serialized messages, as
-follows:
-
-~~~ tls-presentation
-enum {
-  reserved(0), /* Reserved for testing purposes */
-  (255)
-} BatchMode;
-~~~
-
-The time-interval batch mode is described in {{time-interval-batch-mode}}; the
-leader-selected batch mode is described in {{leader-selected-batch-mode}}.
-Future specifications may introduce new batch modes as needed (see
-{{batch-mode-reg}}). Implementations are free to implement only a subset of the
-available batch modes.
-
-A query includes parameters used by the Aggregators to select a batch of
-reports specific to the given batch mode. A query is defined as follows:
-
-~~~ tls-presentation
-struct {
-  BatchMode batch_mode;
-  select (Query.batch_mode) {
-    case time_interval: Interval batch_interval;
-    case leader_selected: Empty;
-  }
-} Query;
-~~~
```

Comment:
Moved to where it's first used, similar to BatchSelector and PartialBatchSelector.